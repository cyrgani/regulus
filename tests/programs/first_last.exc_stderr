IndexError: invalid list index: out of range integral type conversion attempted

IndexError: sequence index out of bounds
at <stl:lists>:13:10
at <stl:lists>:12:18
at programs/first_last.re:6:37
at programs/first_last.re:6:31

IndexError: invalid list index: out of range integral type conversion attempted

IndexError: sequence index out of bounds
at <stl:lists>:13:10
at <stl:lists>:12:18
at programs/first_last.re:8:37
at programs/first_last.re:8:31

