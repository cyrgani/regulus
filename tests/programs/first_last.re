import(lists),
=(l, list(2, 5, 6, "a", list(2, 4))),
assert_eq(first(l), 2),
assert_eq(last(l), list(2, 4)),
__builtin_print_catch(last("")),
__builtin_print_catch(first(list())),
__builtin_print_catch(last(list())),
__builtin_print_catch(first("")),
assert_eq(last("a"), first("a")),
assert_eq(last("a"), "a"),
