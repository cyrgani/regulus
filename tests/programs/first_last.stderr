IndexError: invalid list index: out of range integral type conversion attempted
IndexError: sequence index out of bounds
at <stl:lists>:13:10
at <stl:lists>:12:18
at programs/first_last.re:6:28
at programs/first_last.re:6:22
IndexError: invalid list index: out of range integral type conversion attempted
IndexError: sequence index out of bounds
at <stl:lists>:13:10
at <stl:lists>:12:18
at programs/first_last.re:8:28
at programs/first_last.re:8:22
